vowpalwabbit/v_array.h

```diff
@@ -415,6 +415,7 @@ struct v_array<T, typename std::enable_if<std::is_trivially_copyable<T>::value>:
 };
 
 template <class T>
+VW_DEPRECATED("Use v_array's constructor directly instead. This will be removed in VW 9.0.")
 inline v_array<T> v_init()
```

Comment:
if we were to delete it today with this PR, are all the references in this repo to it gone?

Reply:
yes